setup.py

```diff
@@ -24,6 +24,7 @@ def read_version(package):
         exec(fd.read(), data)
     return data['__version__']
 
+
```

Comment:
flake8 was not happy about the lack of the extra line after the function definition